src/controllers/user.controller.ts

```
@@ -1,7 +1,13 @@
 import { NextFunction, Request, Response, RequestHandler, CookieOptions } from 'express';
 import logger from '@/configs/logger.config';
 import { EmptyResponseDto, LoginResponseDto, UserWithTokenDto } from '@/types';
+import { QRLoginTokenResponseDto } from '@/types/dto/responses/qrResponse.type';
 import { UserService } from '@/services/user.service';
+import { InvalidTokenError, TokenExpiredError } from '@/exception/token.exception';
+import { NotFoundError } from '@/exception';
```

ESLint

[error] 'NotFoundError' is defined but never used.

(@typescript-eslint/no-unused-vars)